Riot/Modules/Room/Views/WYSIWYGInputToolbar/WysiwygInputToolbarView.swift

```diff
@@ -35,6 +35,7 @@ class SelfSizingHostingController<Content>: UIHostingController<Content> where C
 
 @available(iOS 16.0, *)
 class WysiwygInputToolbarView: MXKRoomInputToolbarView, NibLoadable, RoomInputToolbarViewProtocol {
+    @Environment(\.theme) var theme: ThemeSwiftUI
```

Comment:
This is UIKit view so we need to apply the theme in a slightly different way.

Here is an example of how to [apply it during initialisation](https://github.com/vector-im/element-ios/blob/develop/Riot/Modules/ContextMenu/RoomContextPreviewViewController.swift#L63) and how to [have it update when the theme changes](https://github.com/vector-im/element-ios/blob/develop/Riot/Modules/ContextMenu/RoomContextPreviewViewController.swift#L76)

Reply:
done, however I tried to apply the theme first by using a var and the `initialise` function (the one that returns the  `loadFromNib` implementation), but that function seems to no be used so it was crashing the view. I had to set the first theme instance directly into the `awakeFromNib` function